### PR TITLE
CB2-10665: Remove JWT from Log

### PR DESCRIPTION
**Pull request by LGin-BJSS**
## CB2-10665: Remove JWT from Log

Removes JWT token from logs
[link to ticket number](https://dvsa.atlassian.net/browse/CB2-10665)

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/services/tokens.ts [hunk 1]
```diff
@@ -35,7 +33,11 @@ export const getValidJwt = async (authorizationToken: string, logEvent: ILogEven
   }
 
   logEvent.email = username;
+  logEvent.roles = (decoded.payload as JwtPayload).roles;
```
Comment (LGin-BJSS) [inline src/services/tokens.ts:36]: This duplicates what happens in `roles.ts` but is needed here as well to ensure that roles are still logged in the event of other failures and the logs are consistent.

we could remove this line and just have 'undefined' in that scenario. Do we care what the roles of a user are, if that is not the reason the token has failed validation?

Comment (kcarr-bjss) [reply src/services/tokens.ts:36]: I think keeping the roles in there is better for the sake of consistency. Are the roles guaranteed to be populated? If so then I do not see much downside to keeping it.

Comment (LGin-BJSS) [reply src/services/tokens.ts:36]: We can't guarantee anything unfortunately, as an invalid JWT could theoretically contain anything

Comment (kcarr-bjss) [reply src/services/tokens.ts:36]: Perhaps put a null guard in there just to be safe?

Comment (LGin-BJSS) [reply src/services/tokens.ts:36]: I'm not sure if we can guard on it, as if it _is_ null, then we probably want to log out that it is, as that will inform what the issue is when debugging this log.

**Review by cb-cs — APPROVED** [review]

**Review by naathanbrown — APPROVED** [review]

**Comment by naathanbrown** [comment]
Just whilst we are updating this, because we don't come in often, is it worth just checking if we can bump that axios version?

**Review by Daniel-Searle — APPROVED** [review]

**Comment by LGin-BJSS** [comment]
> Just whilst we are updating this, because we don't come in often, is it worth just checking if we can bump that axios version?

Good shout, I've given it a bump now

**Review by kcarr-bjss — APPROVED** [review]

**Pull request merged**